web/app.js

```diff
@@ -1455,11 +1455,6 @@ const PDFViewerApplication = {
             spreadMode,
           });
           this.eventBus.dispatch("documentinit", { source: this });
-          // Make all navigation keys work on document load,
-          // unless the viewer is embedded in a web page.
-          if (!this.isViewerEmbedded) {
-            pdfViewer.focus();
```

Comment:
There is a change in behavior here, and I'm not sure if it's an improvement or a regression (or even intended to change), but while scrolling with PageDown works like before what has changed is the tab order. If you open https://mozilla.github.io/pdf.js/web/viewer.html and hit Tab a couple of times the pages are focused, while if you open http://54.241.84.105:8877/95d19818db46d19/web/viewer.html and hit Tab a couple of times the toolbar buttons are focused instead.

Please check if that is OK before merging this. If it is, r=me, and otherwise please update the patch.

Reply:
Oh I missed that.
Anyway, I think the right thing to do in order to navigate with the keyboard is the following:
 - go the toolbar first element with TAB
 - navigate between toolbar elements in using arrows

It's something we'll have to fix asap.
So for now I think this "regression" is perfectly acceptable since afaik the first focused element should be the toolbar itself.
If we've too much complaints we'll see.